### DISCUSSION TITLE
Follow User

### DISCUSSION THREAD

**Discussion by alzeric** [davidteather/TikTok-Api#530] [Q&A]
Could we get a follow user method or by chance do you have an example of how to do this?

this utilizes the api/commit/follow/user/ endpoint

**Comment by alzeric** [comment]
I feel i'm fairly close getting a 

`[{'status_code': 0, 'status_msg': "url doesn't match"}]`

after running it through getdata

**Reply by BimaSkyy** [reply]
in from 2025 did you find it solution?

**Reply by alzeric** [reply]
Sorry brother, Life happened and got slammed with about 50 other work projects :D
